### ISSUE TITLE
A11y_Azure Data Studio_Nov2023_Filegroups_Add_DevTools: Visual name of "Filegroup name" edit filed is programmatically not associate with edit filed. 

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/azuredatastudio#25128]
**"Please do not close this bug. This bug should only be closed by Trusted Tester after verification"**
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
## Git hub Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #DesktopApp; #WCAG4.1.2; #FTP; #SH_Azure Data Studio_Nov2023_Win32_Dec2023; #SQL Azure Data Studio; #Win32; #DevTools; #NonBenchmark; #NameRoleValue; #Closed;
## Environment Details:
Application Name: Azure Data Studio
Operating System: Win11
Azure data studio 1.47.0
## Repro Steps:
1. Install Azure data studio application and open it
2. Make sure to enable "Preview" feature in settings.
3. Tab till "Database" and press enter to expand the tree.
4. Tab till any database and pressF10 to open context menu.
5. Tab till "Properties Preview" and press enter.
6. Tab till "Filegroups" tab and press enter.
7. Tab till "Add" button and press enter.
8. Tab till "Filegroup name" edit filed and inspect it.
9. You will observe that programmatic name is not given for edit filed. 
## Actual Result:
Visual name of "Filegroup name" edit filed is programmatically not associate with edit filed.
## Expected Result:
Visual name of "Filegroup name" edit filed should be programmatically associate with edit filed.
## User Impact:
Speech input user will not be able to interact with edit filed and screen reader user will not get the edit filed. 
### Attachment:
![25128_A11y_Azure Data Studio_Nov2023_Filegroups_Add_DevTools](https://github.com/microsoft/azuredatastudio/assets/124254981/a8ee5fff-e049-454a-9f0b-8b65c2da549e)

**Comment by kisantia** [comment]
@kapilvaishna this has been fixed. Please install the l[atest insiders build](https://github.com/microsoft/azuredatastudio?tab=readme-ov-file#try-out-the-latest-insiders-build-from-main-branch) and verify the fix.

**Comment by kapilvaishna** [comment]
Issue is not repro in Azure data studio insider (Version 1.48.0) so closing the bug
![Screenshot (996)](https://github.com/microsoft/azuredatastudio/assets/124254981/6625712d-2549-42d9-8f17-dcdcdf580282)
